docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -41,6 +41,13 @@ would like to install the packages to `C:/oe_prereqs`.
 ./scripts/install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1 -DCAPClientType None
 ```
 
+On Windows 10, the Intel PSW is delivered via Windows Update. Running the
+executable installer will fail on Windows 10 machines. To skip PSW installation:
+
+```powershell
+./scripts/install-windows-prereqs.ps1 -LaunchConfiguration SGX1-NoDriver
```

Comment:
Consider including all the options specified in the previous statement (i.e. `-InstallPath C:/oe_prereqs -DCAPClientType None`) since invocations of install-windows-prereqs.ps1 are not going to be additive and folks tend to just copy & paste.